File: level1/5 - Tests et conditions/6 - Traversée du pont.c
//coded by cybereagle2001
#include <stdio.h>

int main()
{
   int premierDe, secondDe;
   scanf("%d %d", &premierDe, &secondDe);
   
   int somme = premierDe + secondDe;
   if (somme >= 10)
   {
      printf("Taxe spéciale !\n");
      printf("%d\n", 36);
   }
   else
   {
      printf("Taxe régulière\n");
      printf("%d\n", 2 * somme);
   }
}

File: java/level 1/5- Tests et conditions/6- Traversée du pont.java
/** coded by cybereagle2001*/

import algorea.Scanner;

class Main{
   static Scanner input= new Scanner(System.in);
   public static void main(String[] args){
      int x= input.nextInt();
      int y= input.nextInt();
      
      if (x+y>=10){
         System.out.println("Taxe spéciale !");
         System.out.println(36);
      }
      else{
         System.out.println("Taxe régulière");
         System.out.println((x+y)*2);
      }
   }
}
